### PR TITLE
Setup initial scaffolding

### PR DESCRIPTION
**Pull request by kyrofa**
Need to request classic confinement and setup CI pipeline, so get started with a snap that doesn't do much.

### CODE REVIEW

**Comment by Arnatious** [comment]
lgtm!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
-# ros-installer
-Simplify the installation of ROS on Ubuntu
+# ROS Installer
+
+[Installing ROS](http://wiki.ros.org/melodic/Installation/Ubuntu) involves a
```
Comment (artivis) [inline README.md]: How about referring to [ROS 2 installation guide](https://index.ros.org/doc/ros2/Installation/Eloquent/Linux-Install-Debians/)?

Comment (kyrofa) [reply README.md]: You mean instead, or in addition to the ROS 1 guide?

Comment (artivis) [reply README.md]: Either way.

**Pull request merged**